CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Added
 - `MaxEncodedLen` trait for items that have a statically known maximum encoded size. ([#268](https://github.com/paritytech/parity-scale-codec/pull/268))
+- `#[codec(crate = <path>)]` top-level attribute to be used with the new `MaxEncodedLen`
```

Comment:
```suggestion
- `#[codec(crate = "<path>")]` top-level attribute to be used with the new `MaxEncodedLen`
```

Right now it expects a string literal instead of a path item. I'm going to be changing that shortly, but for now let's keep the changelog accurate.

Reply:
Makes sense, thanks